<ui version="4.0" >
 <class>PickUserDialogBase</class>
 <widget class="QDialog" name="PickUserDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Select Profile...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item rowspan="2" row="0" column="0" >
    <layout class="QGridLayout" name="gridLayout_2" >
     <item row="0" column="0" colspan="2" >
      <widget class="QLabel" name="TextLabel" >
       <property name="text" >
        <string>Pick one name from the list or enter a new User name/Profile!</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignTop</set>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="TextLabelNewUser" >
       <property name="text" >
        <string>User/Profile:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignTop</set>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="lineEditNewUser" />
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="QPushButton" name="PushButtonOk" >
     <property name="maximumSize" >
      <size>
       <width>59</width>
       <height>25</height>
      </size>
     </property>
     <property name="text" >
      <string>OK</string>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="PushButtonCancel" >
     <property name="maximumSize" >
      <size>
       <width>59</width>
       <height>25</height>
      </size>
     </property>
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="buttonProperties" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>26</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Properties: Rename profile</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="C:/Work/Dev_QT4/Galaxy_Products/gex_product/gex/sources/gex.qrc" >
         <normaloff>:/gex/icons/properties.png</normaloff>:/gex/icons/properties.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemoveUser" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>26</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Remove selected user from list</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="C:/Work/Dev_QT4/Galaxy_Products/gex_product/gex/sources/gex.qrc" >
         <normaloff>:/gex/icons/file_remove.png</normaloff>:/gex/icons/file_remove.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>208</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Users / Profiles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="1" column="0" >
       <widget class="QTreeWidget" name="treeWidgetProfiles" >
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights" >
         <bool>true</bool>
        </property>
        <property name="allColumnsShowFocus" >
         <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Profiles</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Default</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
